the-collab-lab/tcl-79-smart-shopping-list
# Protect routes and create welcome page

## Description


A welcome/login page has been created which is the only page the user will be able to access on the app until they log in. Once they log in they are automatically navigated to the home page and if they log out they are navigated to the login page again.
## Related Issue


Closes #35 
## Acceptance Criteria
- [x] Protect the home and list routes by show a welcoming page with the sign in button if the user isn't signed in. 
- [x] Create the welcoming page following the figma design. 


## Type of Changes

Enhancement

## Updates

### After
<img width="959" alt="image" src="https://github.com/user-attachments/assets/68a630e3-b7af-4986-83b6-f0243d9d00f6">



## Testing Steps / QA Criteria
- Do a `git pull` and `git checkout jo-home-page` 
- If you are signed in then sign out and you will automatically navigate to the login page
- In the url once signed out try putting a `/list` at the end of the localhost to try and see it will allow you to access this page or any of the other two
